.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 .idea
+build
```

Comment:
We'd spoken previously about getting rid of this, going ahead with it since we're cleaning things up anyway.